src/content/docs/infrastructure/host-integrations/host-integrations-list/lightptd-integration.mdx

```diff
@@ -0,0 +1,176 @@
+* If you'd rather install our infrastructure agent manually, you can follow a tutorial for manual installation for [Linux](/docs/infrastructure/install-infrastructure-agent/linux-installation/install-infrastructure-monitoring-agent-linux), [Windows](/docs/infrastructure/install-infrastructure-agent/windows-installation/install-infrastructure-monitoring-agent-windows/), or [macOS](/docs/infrastructure/install-infrastructure-agent/macos-installation/install-infrastructure-monitoring-agent-macos/).
+
+## Integrate Lighttpd with New Relic [#integrate]
+
+Once installed, the infrastructure agent can ingest data from your app and send it to New Relic, but you still need to integrate – or establish a line of communication – between the agent and your app. Once integrated, you can get started with an out of the box monitoring solution for your Lighttpd app. 
+
+<Steps>
+<Step>
+### Manually configure the lighttpd.conf file
+
+To export metrics on the url `127.0.0.1:9880/server-status?format=plain`, use the following steps:
+
+1. Go to lighttpd config file path
+```
+cd /etc/lighttpd/
+```
+2. Open `lighttpd.conf` file to edit
+```
+sudo nano lighttpd.conf
+```
+
+3. Add `mod_status` inside `server.modules` object.
+4. Add status.status-url = `/server-status` after server.modules
+5. Update `server.document-root` with the lighttpd file path `/var/www/html/lighttpd-webpage`
+6. Update `server.port` on which you are running lighttpd. Here, we have updated the server.port to 9880.
+7. Run `127.0.0.1:9880/server-status?auto` or  `127.0.0.1:9880/server-status?format=plain` on the browser to check server related metrics.
+
+```
+server.modules = (
+    "mod_1indexfile",
+    "mod_access”,
+    "mod_alias",
+    "mod_redirect”,
+    "mod_status"
+)
+
+server.document-root	= "/var/www/html/1lighttpd-webpage"
+server.upload-dirs	  = ( "/var/cache/lighttpd/uploads" )
+server.errorlog		    = "/var/log/lighttpd/error.log"
+server.pid-file		    = "/run/lighttpd.pid"
+server.username		    = "www-data"
+server.groupname	    = "www-data"
+server.port		        = 9880
+status.status-url	    = "/server-status”
+```
+</Step>
+
+
+<Step>
+
+## Nri-Flex configuration
+
+First, follow the [installation steps](https://github.com/newrelic/nri-flex/blob/master/docs/basic-tutorial.md) of nri-flex from [nri-flex repository](https://github.com/newrelic/nri-flex)
+
+Once you have installed the infrastructure agent on your host, nri-flex binary is also installed with it.
+
+To create a flex configuration file follow these steps:
+
+1. Path:
+```
+/etc/newrelic-infra/integrations.d
+```
+
+2. Create a file with the name `lighttpd-http-config.yml`
+
+Note: A generic file name could be &ltFILENAME&gt-config.yml
+
+3. Use add the following configuration to your `lighttpd-http-config.yml` file. 
```

Comment:
2. Use add the following configuration to your `lighttpd-http-config.yml` file.

Reply:
This should be 3 because there are two points ahead of it.

Reply:
Nevermind, I understand now.